check.py

```diff
@@ -0,0 +1,2 @@
+def check(i):
```

Comment:
what is this for?

Reply:
This was check file for precommit i removed from git using rm -r -f and i remember push